dashboard/test/controllers/user_school_infos_controller_test.rb

```diff
@@ -0,0 +1,32 @@
+require 'test_helper'
+
+class UserSchoolInfosControllerTest < ActionDispatch::IntegrationTest
+  test "last confirmation date in user school infos table is updated" do
+    user_school_info = create :user_school_info
+    sign_in user_school_info.user
+    patch "/api/v1/user_school_infos/#{user_school_info.id}/update_last_confirmation_date/"
+    assert_response :success
```

Comment:
these tests should all also verify that the actual data has been updated, not just that the endpoint reported success